article/app/views/articleAMP.scala.html

```diff
@@ -1,5 +1,7 @@
 @(model: ArticlePage)(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
 
 @mainAMP(model, model.related, model.article.content){
+
+}{
```

Comment:
non-liveblog articles do not receive any structured data so we pass through nothing in this case.